sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2703,6 +2703,11 @@ pi_result cuda_piEventGetProfilingInfo(pi_event event,
 
   assert(event != nullptr);
 
+  pi_queue queue = event->get_queue();
+  if (queue == nullptr || !(queue->properties_ & PI_QUEUE_PROFILING_ENABLE)) {
```

Comment:
are you sure we should return this error when `queue` is null? what is the event without an associated queue?

Reply:
Only user events do not have queues associated with them - and while we do not support user events, this is the error code returned when trying to get performance data from one.